.github/workflows/ci.yml

```diff
@@ -22,6 +22,11 @@ jobs:
         with:
           python-version: '3.8'  # Specify the Python version you need
 
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install numpy pandas pytest
+  
       - name: Run tests
         run: |
-          python -m unittest discover -s tests
\ No newline at end of file
+          python -m unittest discover -s tests
```

Comment:
what happens if this line turns to plain old `pytest`?

Reply:
it will have the same end result, I believe the current version just gives less details about passed tests but can run slightly faster